examples/dotnet/AuthInitialize/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.8.2)
+
+project(AuthInitialize CSharp)
+
+message(STATUS "Assembly is ${GEODE_ASSEMBLY}")
+
+if ("${GEODE_ASSEMBLY}" STREQUAL "")
```

Comment:
Should be using the `find_package` to find this and not force the user to specify it.

Reply:
The find_package call in Findgeode-native.cmake is broken and we made the decision to just require the location explicitly rather than rathole on fixing the script for this story.